### PR TITLE
GitHub setup

### PR DESCRIPTION
**Pull request by GeckoSplinter**
That will make TLS monitor publicly available while having the image in our organisation

+ update on chart from @etifontaine 
+ update on deps

### CODE REVIEW

File: .github/workflows/docker-image.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```
Comment (usmonster) [inline .github/workflows/docker-image.yaml:22]: Why not v3?
```suggestion
        uses: actions/checkout@v3
```

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+version: 2
```
Comment (OuesFa) [inline .github/dependabot.yaml:1]: Shouldn't we use Renovate to update the deps instead?

Comment (GeckoSplinter) [reply .github/dependabot.yaml:1]: Nope because it's a public repo

Comment (jhaumont) [reply .github/dependabot.yaml:1]: you're right. I don't remember why I said to @GeckoSplinter to use dependabot

Comment (OuesFa) [reply .github/dependabot.yaml:1]: I guess Julien you didn't see Thomas's answer above. 
I'm resolving this, thanks!

Comment (GeckoSplinter) [reply .github/dependabot.yaml:1]: Anyway we can easily change it after if needed :)

**Pull request merged**